certbot-ci/certbot_integration_tests/utils/pebble_artifacts.py

```diff
@@ -49,6 +50,22 @@ def _fetch_asset(asset: str, assets_path: str) -> str:
     return asset_path
 
 
+def _get_validated_os_and_architecture() -> Tuple[str, str]:
```

Comment:
```suggestion
def _get_validated_os_and_architecture() -> tuple[str, str]:
```

To make this independent of https://github.com/certbot/certbot/pull/10215/files, which removes the `Tuple` import.